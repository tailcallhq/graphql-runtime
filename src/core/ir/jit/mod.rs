mod blueprint_index;
mod builder;
mod model;
