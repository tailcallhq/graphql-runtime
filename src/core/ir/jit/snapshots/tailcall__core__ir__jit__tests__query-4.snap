---
source: src/core/ir/jit/mod.rs
expression: "format!(\"{:#?}\", q_blueprint)"
---
QueryPlan {
    fields: [],
}
